.copier-answers.yml

```diff
@@ -1,5 +1,5 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.19
+_commit: v1.20-4-g56d206b
```

Comment:
Why it is not juste 1.20 here ?

Reply:
Hello @legalsylvain, I updated as you suggest :pray:

Reply:
Hi @legalsylvain it can happen that some changes have been merged in template while a new version has not been released; that was the case here, updating repo with template v1.20, but commit 56d206b
anyway this PR can now be closed as the repo as been [mass-updated meanwhile for another reason](https://github.com/OCA/oca-addons-repo-template/issues/250) :)